decidim-admin/app/controllers/decidim/admin/moderations_controller.rb

```diff
@@ -67,6 +67,10 @@ def unhide
 
       private
 
+      def ransack_params
+        query_params[:q] || { s: "created_at desc" }
```

Comment:
Although I personally think we should use `created_at` default sorting on most of the admin tables, in this particular case of moderation I think we should use the `count` column: as an admin I want to see the content that has received more reports first. I'd love to hear @carolromero opinion on this subject also.

Reply:
We have the following behavior from the moderation team : They visit the website and sometimes they do report some resources, and after that, they visit the Global moderation panel, and they try to hide the reported elements. Ordering by count, the reported resources will get scattered between other reported elements, making it even harder for the moderators team to filter the content.

Reply:
Discussed it with @decidim/product - we agree that the best flow is the one that you propose 👍🏽